usr/lib/tik/modules/post/15-encrypt

```diff
@@ -196,14 +196,18 @@ add_key() {
         while true
         do
             retval=0
-            key="$(zenity --password --title='Set Encryption Passphrase')" || retval=$?
+            if $gui; then
+                key="$(zenity --password --title='Set Encryption Passphrase')" || retval=$?
+            else
+                cenity key --password --title="Set Encryption Passphrase" || retval=$?
+            fi
             case $retval in
                 0)
                     prun /usr/sbin/cryptsetup luksAddKey --key-file=${tik_keyfile} --batch-mode --force-password "${cryptpart}" <<<"${key}"
                     return 0
                 ;;
                 1|255)
-                    zenity --question --text="Do you really want to quit?" && exit 1
+                    d --question --text="Do you really want to quit?" && exit 1
```

Comment:
This ends up being a forced abort of the install because if you click "No", informing the installer you dont want to exit, you end up triggering `d`'s error handling that makes the installer think you want to exit.. :)

We really need that `d-opt` function mentioned in other comments :)